cpp/ql/lib/upgrades/a5bb28ed29f73855d64cc5f939cef977fa8fd19a/upgrade.properties

```diff
@@ -0,0 +1,4 @@
+description: Support repeated initializers, which are allowed in C with designated initializers.
+compatibility: full
```

Comment:
Shouldn't this be backwards? Or will the old database also contain all the tuples for repeated initialisers (it's not clear to me what the database will look like when there are dataset check errors)?

Reply:
The old database contains the repeated initializers (with dataset check errors). The upgrade script keeps and orders them correctly, so we should have exactly the same table as we'd get with the new extractor.